src/App.jsx

```diff
@@ -1,12 +1,30 @@
-import { useState } from 'react'
-import './App.css'
-import movieData from './mockData'
+import { useState, useEffect } from 'react'
 import AllMovies from './components/AllMovies/AllMovies'
 import SingleMovie from './components/SingleMovie/SingleMovie'
 
 export default function App() {
-  const [movies, setMovies] = useState(movieData.movies)
+  const [movies, setMovies] = useState([])
   const [selectedSingleMovie, setSelectedSingleMovie] = useState(null)
+  const [error, setError] = useState(null)
+
+  const fetchMovieData = async () => {
+    const url = 'https://rancid-tomatillos.herokuapp.com/api/v2/movies'
+    try {
+      const response = await fetch(url)
+
+      if (response.status === 500) {
```

Comment:
Consider if it would be valuable to rely on the response.ok data point here rather than a specific hard coded response status.  Its up to you, you'll just want to be able to speak to why you've decided one way vs the other.

Reply:
As you continue to refactor, consider pulling the fetch functionality into an apiCalls file